cli/src/app/service/configuration/SetConfigurationService.ts

```diff
@@ -378,54 +406,232 @@ export default class SetConfigurationService extends Service {
+      language.getText('configuration.fireblocks.askVaultAccountId'),
+      '2',
+    );
+    const hederaAccountPublicKey = await this.askForHederaAccountPublicKey(
+      'configuration.askAccountPubKey',
+      '04eb152576e3af4dccbabda7026b85d8fdc0ad3f18f26540e42ac71a08e21623',
+    );
 
     return {
-      accountId: accountId,
-      privateKey: { key: privateKey, type: pkType },
-      network,
-      alias: alias,
+      apiSecretKey,
+      apiKey,
+      baseUrl,
+      assetId,
+      vaultAccountId,
+      hederaAccountPublicKey,
     };
   }
 
+  public async askForDfnsOfAccount(): Promise<IDfnsAccountConfig> {
+    utilsService.showMessage(language.getText('configuration.dfns.title'));
+    const authorizationToken = await utilsService.defaultPasswordAsk(
+      language.getText('configuration.dfns.askAuthorizationToken'),
+    );
+    const credentialId = await this.askForDfnsCredentialId();
+    const privateKey = await this.askForDfnsPrivateKey();
+    const appOrigin = await this.askForUrl(
+      'configuration.dfns.askAppOrigin',
+      'https://localhost:3000',
+    );
+    const appId = await this.askForDfnsAppId();
+    const testUrl = await this.askForUrl(
+      'configuration.dfns.askTestUrl',
+      'https://api.dfns.ninja/',
+    );
+    const walletId = await this.askForDfnsWalletId();
+    const hederaAccountPublicKey = await this.askForHederaAccountPublicKey(
+      'configuration.askAccountPubKey',
+      '04eb152576e3af4dccbabda7026b85d8fdc0ad3f18f26540e42ac71a08e21623',
+    );
+
+    return {
+      authorizationToken,
+      credentialId,
+      privateKey,
+      appOrigin,
+      appId,
+      testUrl,
+      walletId,
+      hederaAccountPublicKey,
+    };
+  }
+
+  private async askForDfnsCredentialId(): Promise<string> {
+    let credentialId = '';
+    const credentialIdRegExpValidator = /[a-zA-Z0-9]{42}$/g;
+    while (!credentialIdRegExpValidator.test(credentialId)) {
+      credentialId = await utilsService.defaultSingleAsk(
+        language.getText('configuration.dfns.askCredentialId'),
+        'Y2ktMTZ2NTMtZXF0ZzAtOWRvOXJub3NjbGI1a3RwYg',
+      );
+    }
+    return credentialId;
+  }
+
+  private async askForDfnsPrivateKey(): Promise<string> {
+    let privateKey = '';
+    const privateKeyRegExpValidator =
+      /-----BEGIN EC PRIVATE KEY-----[a-zA-Z0-9+/=\n]+-----END EC PRIVATE KEY-----/gm;
+
+    while (!privateKeyRegExpValidator.test(privateKey)) {
+      privateKey = await utilsService.defaultPasswordAsk(
+        language.getText('configuration.dfns.askPrivateKey'),
+      );
+    }
+
+    return privateKey;
+  }
+
+  private async askForDfnsAppId(): Promise<string> {
+    let appId = '';
+    const appIdRegExpValidator = /ap-[a-z0-9]+-[a-z0-9]+-[a-z0-9]+/g;
+    while (!appIdRegExpValidator.test(appId)) {
+      appId = await utilsService.defaultSingleAsk(
+        language.getText('configuration.dfns.askAppId'),
+        'ap-2ng9jv-80cfc-983pop0iauf2sv8r',
```

Comment:
Should we delete the default values?

Reply:
I set it to have a better user experience. I will modify it a bit maintaining the format.